File: src/main/java/com/gmail/borlandlp/minigamesdtools/arena/team/TeamCreatorHub.java
package com.gmail.borlandlp.minigamesdtools.arena.team;

import com.gmail.borlandlp.minigamesdtools.creator.AbstractDataProvider;
import com.gmail.borlandlp.minigamesdtools.creator.CreatorHub;

public class TeamCreatorHub extends CreatorHub {
    public TeamProvider createTeam(String ID, AbstractDataProvider dataProvider) throws Exception {
        return (TeamProvider) this.create(ID, dataProvider);
    }
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/arena/team/TeamCreatorHub.kt
package com.gmail.borlandlp.minigamesdtools.arena.team

import com.gmail.borlandlp.minigamesdtools.creator.AbstractDataProvider
import com.gmail.borlandlp.minigamesdtools.creator.CreatorHub

class TeamCreatorHub : CreatorHub() {
    @Throws(Exception::class)
    override fun create(itemID: String, dataProvider: AbstractDataProvider): TeamProvider {
        return super.create(itemID, dataProvider) as TeamProvider
    }
}